agentops/llm_tracker.py

```diff
@@ -610,3 +707,10 @@ def undo_override_openai_v1_async_completion(self):
         from openai.resources.chat import completions
 
         completions.AsyncCompletions.create = original_create_async
+
+    def undo_override_ollama(self):
```

Comment:
Can you the undo_override_ollama to the stop_instrumenting function at line 695?